packages/leafygreen-provider/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import LeafyGreenProvider, { UsingKeyboardContext } from './LeafyGreenProvider';
+import useShowFocus from './useShowFocus';
+
+export { UsingKeyboardContext, useShowFocus };
```

Comment:
Can we not export the context publicly? That shouldn't be externally accessible. We don't want people to directly interface with the context outside of our control.